### PR TITLE
PIDs limit  and Maximum Log size setting is already deprecated in ContainerRuntimeConfig

### PR DESCRIPTION
**Pull request by PoornimaSingour**
Version(s):

4.12, 4.13,4.14

Issue:

NA

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@haircommander, will you please confirm this change?

**Comment by haircommander** [comment]
sorry for the delay, LGTM

**Comment by lahinson** [comment]
Preview link: https://65986--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks#create-a-containerruntimeconfig_post-install-machine-configuration-tasks

File: modules/create-a-containerruntimeconfig-crd.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ To revert the changes implemented by using a `ContainerRuntimeConfig` CR, you mu
 
 You can modify the following settings by using a `ContainerRuntimeConfig` CR:
 
-* **PIDs limit**: Setting the PIDs limit in the `ContainerRuntimeConfig` is expected to be deprecated. If PIDs limits are required, it is recommended to use the `podPidsLimit` field in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
+* **PIDs limit**: Setting the PIDs limit in the `ContainerRuntimeConfig` is deprecated. If PIDs limits are required, it is recommended to use the `podPidsLimit` field in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```
Comment (lahinson) [inline modules/create-a-containerruntimeconfig-crd.adoc:18]:
```suggestion
* **PIDs limit**: Setting the PIDs limit in the `ContainerRuntimeConfig` CR is deprecated. If PIDs limits are required, use the `podPidsLimit` field in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```
- Per IBM Style, the term "recommend" is a use-with-caution word and should be avoided if possible. See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r.

File: modules/create-a-containerruntimeconfig-crd.adoc [hunk 2]
```diff
@@ -24,7 +24,7 @@ The CRI-O flag is applied on the cgroup of the container, while the Kubelet flag
 
 * **Log level**: The `logLevel` parameter sets the CRI-O `log_level` parameter, which is the level of verbosity for log messages. The default is `info` (`log_level = info`). Other options include `fatal`, `panic`, `error`, `warn`, `debug`, and `trace`.
 * **Overlay size**: The `overlaySize` parameter sets the CRI-O Overlay storage driver `size` parameter, which is the maximum size of a container image.
-* **Maximum log size**: Setting the maximum log size in the `ContainerRuntimeConfig` is expected to be deprecated. If a maximum log size is required, it is recommended to use the `containerLogMaxSize` field in the `KubeletConfig` CR instead.
+* **Maximum log size**: Setting the maximum log size in the `ContainerRuntimeConfig` is deprecated. If a maximum log size is required, it is recommended to use the `containerLogMaxSize` field in the `KubeletConfig` CR instead.
```
Comment (lahinson) [inline modules/create-a-containerruntimeconfig-crd.adoc:27]:
```suggestion
* **Maximum log size**: Setting the maximum log size in the `ContainerRuntimeConfig` CR is deprecated. If a maximum log size is required, use the `containerLogMaxSize` field in the `KubeletConfig` CR instead.
```

**Review by lahinson — COMMENTED** [review]
@PoornimaSingour I made a couple of suggestions for your consideration, but overall, LGTM.

**Comment by lahinson** [comment]
@PoornimaSingour Also, please be sure to select the QE checkbox in the first comment of this PR if QE has approved this PR. Thanks!

**Comment by PoornimaSingour** [comment]
There is no clear due date at the moment as this feature is coordinated with the upstream Kubernetes project to be removed, hence we can not do the doc change at the moment, hence closing this PR

**Pull request closed without merging**